test/functional/tests/security/test_security_io_engines.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright(c) 2019-2020 Intel Corporation
+# Copyright(c) 2020 Intel Corporation
```

Comment:
It's a good practice to avoid having commits that fix problems introduced by other commits within the same pull request. Please squash those two commits into one.